### PR TITLE
Install and setup PostGres, NewRelic & Artillery

### PR DESCRIPTION
**Pull request by andrewdeanyoder**

### CODE REVIEW

File: database/postgres/db_postgres_benchmark.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+if (!joinedQuery) {
+  query = `SELECT * FROM reviews WHERE review_id=${reviewID}`;
+} else {
+  query = `SELECT * FROM reviews, authors, hostels
+  WHERE reviews.review_id=${reviewID}
+  AND reviews.author_id = authors.id
+  AND reviews.hostel_id = hostels.id`;
+}
```
Comment (miraclestyle) [inline database/postgres/db_postgres_benchmark.js]: Could this code be wrapped in a function?

File: database/postgres/db_postgres_benchmark.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+let query;
+if (!sameRecord) {
+  const last90 = Math.floor(records * 0.9);
+  reviewID = getRandomInt(last90, records);
+} else {
+  reviewID = Math.floor(records * 0.9);
+}
+
+if (!joinedQuery) {
+  query = `SELECT * FROM reviews WHERE review_id=${reviewID}`;
```
Comment (miraclestyle) [inline database/postgres/db_postgres_benchmark.js]: Directly manipulating query string with external variables poses security vulnerability. Instead, consider passing the values in an array as the second positional argument of the query method.

File: database/postgres/postgres.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    hostel_id, author_id, description, security, location,
+    staff, atmosphere, cleanliness, facilities, value, total,
+  } = body;
+
+  // get the current timestamp and format it
+  const timestamp = Date.now();
+  let createdAt = new Date(timestamp);
+  createdAt = moment(createdAt).format('YYYY-MM-DD');
+
+  const queryStr = `INSERT INTO reviews (hostel_id, author_id, description, date, security, location, staff, atmosphere, cleanliness, facilities, value, total) VALUES (${hostel_id}, ${author_id}, '${description}', '${createdAt}', ${security}, ${location}, ${staff}, ${atmosphere}, ${cleanliness}, ${facilities}, ${value}, ${total})`;
```
Comment (miraclestyle) [inline database/postgres/postgres.js]: Line formatting issues are universal for all of us sharing the same VSCode configuration.

File: database/postgres/postgres.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  cleanliness FLOAT(1) NOT NULL,
+  facilities FLOAT(1) NOT NULL,
+  value FLOAT(1) NOT NULL,
+  total FLOAT(1) NOT NULL,
+  FOREIGN KEY (hostel_id) REFERENCES hostels(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id)
+);
+
+COPY hostels
+FROM '/Users/andrew/Documents/HackReactor/SDC/Reviews-Service/database/data/hostels.csv'
```
Comment (miraclestyle) [inline database/postgres/postgres.sql:41]: How will this path be recognized on an EC2 instance?

**Review by miraclestyle — APPROVED** [review]
You may want to take a closer look at one comment here. Everything else is good.

File: database/postgres/postgres.sql (hunk 4 above)
Comment (andrewdeanyoder) [reply database/postgres/postgres.sql:41]: Good point, I'll have to change it on the instance.

File: database/postgres/db_postgres_benchmark.js (hunk 1 above)
Comment (andrewdeanyoder) [reply database/postgres/db_postgres_benchmark.js]: Yes, this block would be a great candidate for a function if I were reusing it anywhere.

**Pull request merged**